.circleci/test-deploy.yml

```diff
@@ -84,6 +84,17 @@ jobs:
       - python/dist:
           build-tool: << parameters.build-tool >>
           app-dir: ~/project/sample_pip
+  dist-test-build-opts:
```

Comment:
The job definition is here, but it's never called in the workflow. Maybe you forgot to include it?